jupyterfs/tests/test_fsmanager.py

```diff
@@ -131,11 +131,14 @@ def setup_method(self, method):
     def teardown_method(self, method):
         shutil.rmtree(self._test_dir, ignore_errors=True)
 
-    @pytest.fixture
-    def resource_uri(self, tmp_path):
-        yield f"osfs://{tmp_path}"
+    def get_resource(self, resource_type, tmp_path=None):
+        if resource_type == "pyfs":
+            return {"url": f"osfs://{tmp_path}", "type": resource_type}
+        elif resource_type == "fsspec":
+            return {"url": f"file://{tmp_path}", "type": resource_type}
 
 
+@pytest.mark.skipif(which("docker") is None, reason="requires docker")
```

Comment:
see note above, tested locally on mac against docker but cannot use docker in arm macs on GHA.